WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -61,6 +61,10 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         super.viewDidAppear(animated)
 
         workaroundLargeTitleCollapseBug()
+        guard self.children.contains(where: {$0 is BlogDetailsViewController}) else {
```

Comment:
If we do keep this, it might be good to move it into a small helper method – `isShowingBlogDetails` or something? And I think we can just check if `blogDetailsViewController == nil`, as the comment on that property states that it should only exist if we're showing blog details.

So we'd end up with

```
if isShowingBlogDetails {
    WPTabBarController.sharedInstance()?.presentWhatIsNew(on: self)
}
```